tests/basics/sys_stdio.py

```diff
@@ -0,0 +1,33 @@
+# Test sys.std* objects.
+
+import sys
+
+try:
+    sys.stdout
+    sys.stdin
+    sys.stderr
+except AttributeError:
+    print("SKIP")
+    raise SystemExit
+
+# CPython is more verbose; no need to match exactly
+
+print('TextIOWrapper' in str(sys.stdout))
+print('TextIOWrapper' in str(sys.stderr))
+print('TextIOWrapper' in str(sys.stdin))
+
+print('TextIOWrapper' in str(type(sys.stdout)))
+print('TextIOWrapper' in str(type(sys.stderr)))
+print('TextIOWrapper' in str(type(sys.stdin)))
+
+# # .buffer member is optional
```

Comment:
I suggest putting the following code (uncommented) into a separate test `tests/basics/sys_stdio_buffer.py`, and skip that test by first checking if `sys.stdout.buffer` exists (like this file does for `sys.stdout`).

Reply:
You can also add a `tests/basics/sys_stdio_buffer.py.exp` file if the output doesn't match CPython.

Reply:
> separate test `tests/basics/sys_stdio_buffer.py`
Done.

> add a `tests/basics/sys_stdio_buffer.py.exp` file if the output doesn't match CPython
A weak match to CPython is appropriate for this.